__test__/index.test.js

```diff
@@ -1,7 +1,12 @@
+const puppeteer = require('puppeteer');
 const timeout = 25000;
 
 describe('Dummy Test ', () => {
   beforeAll(async () => {
+    browser = await puppeteer.launch({
+      headless: 'new',
+    });
+    page = await browser.newPage();
```

Comment:
This change is for the warning we discussed on call right? Or this was also affecting the test fail?

Reply:
- Yes, This is required as we are trying to access the `page` instance. Else it would an error stating page isn`t defined while we try to hit the rds url. 
 `    await page.goto('https://www.realdevsquad.com/', {
      waitUntil: 'domcontentloaded',
    });` 

- As discussed on the call, the `{
      headless: 'new',
    }` being passed over while launching puppeteer is for the warning.